plugin-flex-ts-template-v2/src/utils/feature-loader/logger.ts

```diff
@@ -24,5 +20,6 @@ export const addHook = (feature: string, hook: any) => {
     logger.warn(`Feature ${feature} declared logger hook, but is missing loggerHook to hook`);
     return;
   }
+  console.info(`Feature ${feature} registered logger hook: %c${typeof hook.loggerHook}`, 'font-weight:bold');
```

Comment:
I'd like to go for consistency here and log the function name if possible! You'll need to adjust the hook to have a function name as well.